src/wp-includes/deprecated.php

```diff
@@ -4225,3 +4225,63 @@ function wp_render_duotone_filter_preset( $preset ) {
 	_deprecated_function( __FUNCTION__, '5.9.1', 'wp_get_duotone_filter_property()' );
 	return wp_get_duotone_filter_property( $preset );
 }
+
+/**
+ * Checks whether serialization of the current block's border properties should
+ * occur.
+ *
+ * @since 5.8.0
+ * @access private
+ * @deprecated 6.0.0 Use `wp_should_skip_block_supports_serialization` introduced in 6.0.0.
+ *
+ * @param WP_Block_Type $block_type Block type.
+ * @return bool Whether serialization of the current block's border properties
+ *              should occur.
+ */
+function wp_skip_border_serialization( $block_type ) {
+	_deprecated_function( __FUNCTION__, '6.0.0', 'wp_should_skip_block_supports_serialization()' );
+	$border_support = _wp_array_get( $block_type->supports, array( '__experimentalBorder' ), false );
+
+	return is_array( $border_support ) &&
+		array_key_exists( '__experimentalSkipSerialization', $border_support ) &&
+		$border_support['__experimentalSkipSerialization'];
+}
+
+/**
+ * Checks whether serialization of the current block's dimensions properties
+ * should occur.
+ *
+ * @since 5.9.0
+ * @access private
+ * @deprecated 6.0.0 Use `wp_should_skip_block_supports_serialization` introduced in 6.0.0.
+ *
+ * @param WP_Block_type $block_type Block type.
+ * @return bool Whether to serialize spacing support styles & classes.
+ */
+function wp_skip_dimensions_serialization( $block_type ) {
+	_deprecated_function( __FUNCTION__, '6.0.0', 'wp_should_skip_block_supports_serialization()' );
+	$dimensions_support = _wp_array_get( $block_type->supports, array( '__experimentalDimensions' ), false );
+	return is_array( $dimensions_support ) &&
+		array_key_exists( '__experimentalSkipSerialization', $dimensions_support ) &&
+		$dimensions_support['__experimentalSkipSerialization'];
+}
```

Comment:
Nitpick: is there a reason not to use `empty`?
```suggestion
	return is_array( $dimensions_support ) && ! empty ( $dimensions_support['__experimentalSkipSerialization'] );
```

Reply:
As [above](https://github.com/WordPress/wordpress-develop/pull/2522#discussion_r846855936)